ansible/playbooks/nagios/roles/Nagios_Config/scripts/Create_Nagios_Server_Configurations.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import json
+import os
+import subprocess
+import sys
+from os import path
+
+import yaml
+try:
+    import configparser
+except ImportError:
+    import ConfigParser as configparser
+
+valid_types = ('build', 'dockerhost', 'test')
+
+Input_Path = sys.argv[1]
+Output_Path = sys.argv[2]
+
+def main():
+
+    # load config file for special cases
+    config = configparser.RawConfigParser()
+    config.read('ansible.cfg')
+
+    # load public inventory
+    basepath = path.dirname(__file__)
+    inventory_path = "{Input_Path}"
```

Comment:
This fails for me `FileNotFoundError: [Errno 2] No such file or directory: '{Input_Path}'`
Change it to 
```suggestion
    inventory_path = Input_Path
```
Or change the line below to 
`export = parse_yaml(load_yaml_file(Input_Path), config)`